### ISSUE TITLE
Integration not loaded on HA 2024.12.0b0

### ISSUE THREAD

**Issue by piitaya** [fuatakgun/eufy_security#1258]
**Describe the bug**

The integration doesn't work with HA 2024.12 beta.

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 1071, in _get_component
    ComponentProtocol, importlib.import_module(self.pkg_path)
                       ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/util/loop.py", line 200, in protected_loop_func
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1022, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/config/custom_components/eufy_security/__init__.py", line 13, in <module>
    from .const import COORDINATOR, DOMAIN, PLATFORMS
  File "/config/custom_components/eufy_security/const.py", line 16, in <module>
    from .model import EntityDescription
  File "/config/custom_components/eufy_security/model.py", line 19, in <module>
    class ConfigField(Enum):
    ...<14 lines>...
        mfa_input = auto()
  File "/config/custom_components/eufy_security/model.py", line 25, in ConfigField
    rtsp_server_address = auto()
    ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.13/enum.py", line 431, in __setitem__
    v.value = self._generate_next_value(
              ~~~~~~~~~~~~~~~~~~~~~~~~~^
            key, 1, len(self._member_names), self._last_values[:],
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
            )
            ^
  File "/usr/local/lib/python3.13/enum.py", line 1248, in _generate_next_value_
    raise TypeError('unable to sort non-numeric values') from None
TypeError: unable to sort non-numeric values
```

**To reproduce**

Update to HA 2024.12 beta.

**Expected behavior**

**Additional information**

Go to Settings -> System -> Repairs -> Click on 3 dots (...) -> System Information and get Version and Installation Type to below fields;
- Home Assistant Installation Type (OS - Supvervised - Core): 
- Home Assistant Core Version: 
- Eufy Security Add-on Version: 
- Eufy Security Integration Version:
- Using go2rtc Add-on or webrtc custom integration with integrated go2rtc: 

Hardware Information;
- Camera Model:

- Live Streaming Protocol (RTSP/P2P): 

- Debug Logs from Home Assistant (https://github.com/fuatakgun/eufy_security/discussions/624) - Without this, I will automatically resolve this issue
  

- Logs from Eufy Security Add-on (either from Add-ons page of Home Assistant or Docker output):

- Would you share your device with me in case further debugging required? (Yes/No):

**Additional context**

Add any other context about the problem goes here. Mainly, share everything as log files and use screenshots as last resort.

**Comment by Josh4532** [comment]
can also confirm the fix does infact fix the issue, even up to 2024.12.0b3

**Comment by BillyFKidney** [comment]
The issue exists with the GA release as well:

**Core**
2024.12.0

**Supervisor**
2024.11.4

**Operating System**
14.0

**Frontend**
20241106.2

<img width="1800" alt="image" src="https://github.com/user-attachments/assets/c94f2476-c73c-458e-93d1-f7d239aade6e">

**Comment by BillyFKidney** [comment]
Can confirm the issue is fixed in v8.1.1

**Comment by R-ens** [comment]
For me it's sill not working. Integration is not loading somehow.

- No logs
- Unable to add new Eufy Security device: "Only a single instance is allowed."

What's going on? Tried to downgrade both HA Core and Eufy Security already, without any success.
